packages/admin/admin/src/router/PromptHandler.test.tsx

```diff
@@ -51,6 +52,62 @@ test("Nested route in Prompt", async () => {
     });
 });
 
+test("Nested dynamic route in Prompt", async () => {
+    function Aasdf() {
+        const subRoutePrefix = useSubRoutePrefix();
+        return (
+            <RouterPrompt
+                message={() => {
+                    return "sure?";
+                }}
+                subRoutePath={`${subRoutePrefix}/s`}
+            >
+                <Link to={`${subRoutePrefix}/s/sub`}>subLink</Link>
+                <Link to={`${subRoutePrefix}`}>back</Link>
+                <Route path={`${subRoutePrefix}/s/sub`}>
+                    <div>sub</div>
+                </Route>
+            </RouterPrompt>
+        );
+    }
+    function Story() {
+        return (
+            <Switch>
+                <Route path="/foo/:param">
+                    <Aasdf />
+                </Route>
+                <Redirect to="/foo/paramvalue" />
+            </Switch>
+        );
+    }
+
+    const rendered = render(
```

Comment:
Please use the `render` function introduced in #1441, which allows you to remove these default providers